api_app/models/domain/resource_template.py

```diff
@@ -23,6 +23,7 @@ class Property(AzureTREModel):
     pattern: Optional[str] = Field(None, title="Pattern")
     updateable: Optional[bool] = Field(None, title="Indicates that the field can be updated")
     readOnly: Optional[bool] = Field(None, title="Indicates the field is read-only")
+    items: Optional[dict] = None
```

Comment:
Why is this a `None`, and not a `Field(None, title="...")`? 
Also, if it has to be `None`, could you add a comment indicating what this is (since other fields have a `title` field that serves this purpose)